packages/http-server-javascript/.gitignore

```diff
@@ -0,0 +1 @@
+src/helpers
```

Comment:
what is this for?

Reply:
It prevents the generated helper modules from build-helpers.ts from being included as changes. I don't want them checked in, just created when the program is built.

Reply:
I would say it might be better to keep those in a separate folder than `src/` to make it clear those are generated.
Most package with decorators have a `_generated` folder where decorator signature is generated, could maybe reuse that